maps/away/ships/heph/cyclops/cyclops_ghostroles.dm

```diff
@@ -0,0 +1,106 @@
+//Cyclops Crew
+/datum/ghostspawner/human/cyclops_crew
+	short_name = "cyclops_crew"
+	name = "Hephaestus Cyclops Mining Crew"
+	desc = "Crew the Hephaestus Mining Vessel."
+	tags = list("External")
```

Comment:
`welcome_message` should be set on all new ghostspawners as discussed here <https://github.com/Aurorastation/Aurora.3/pull/16248#issuecomment-1521304132>